cypress/e2e/project-list.cy.ts

```diff
@@ -2,6 +2,18 @@ import capitalize from "lodash/capitalize";
 import mockProjects from "../fixtures/projects.json";
 
 describe("Project List", () => {
+  const testLoadingImage = () => {
```

Comment:
Why did you assign the test to a variable? Haven't seen this before. I'd remove this and simply keep the `it` statement. This way the test is run once in this test suite which should be sufficient.

Reply:
Ah now I see, this is to not duplicate the test for desktop and mobile, right? To be honest, I'd either test this only for one viewport or duplicate the code